### PR TITLE
Update task-scheduling.md

### PR DESCRIPTION
**Pull request by manoj-manoharan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/techniques/task-scheduling.md [hunk 1]
```diff
@@ -52,6 +52,23 @@ export class TasksService {
 }
 ```
 
+Register the service in AppModule
```
Comment (micalevisk) [inline content/techniques/task-scheduling.md:55]:
```suggestion
Register the service in `AppModule`
```

File: content/techniques/task-scheduling.md [hunk 2]
```diff
@@ -52,6 +52,23 @@ export class TasksService {
...
+@Module({
+  imports: [
+    ScheduleModule.forRoot()
+  ],
+  providers: [TaskService],
+})
+export class AppModule {}
+```
+
+
```
Comment (Tony133) [inline content/techniques/task-scheduling.md:71]:
```suggestion
```

**Review by Tony133 — CHANGES_REQUESTED** [review]
There is an extra space between the text and the code above, for the rest it's fine. 👍

**Comment by kamilmysliwiec** [comment]
This should be obvious to anyone who at least walked through the "Overview" category which is essential to understanding later chapters.

**Pull request closed without merging**